test/integration/cli/generate_test.rb

```diff
@@ -0,0 +1,103 @@
+require 'test_helper'
+
+describe 'lotus generate' do
+  describe 'action' do
+    let(:options)           { '' }
+    let(:app_name)          { 'web' }
+    let(:template_engine)   { 'erb' }
+    let(:framework_testing) { 'minitest' }
+
+    def create_temporary_dir
+      @tmp = Pathname.new(@pwd = Dir.pwd).join('tmp/integration/cli/generate')
+      @tmp.rmtree if @tmp.exist?
+      @tmp.mkpath
+
+      Dir.chdir(@tmp)
+    end
+
+    def generate_application
+      `bundle exec lotus new #{ @app_name = 'delivery' }#{ options }`
+      Dir.chdir(@root = @tmp.join(@app_name))
+
+      File.open(@root.join('.lotusrc'), 'w') do |f|
```

Comment:
@AlfonsoUceda Because it's doesn't override already existing files. In this test case it already exist because we have just generated a fresh app.